pkg/logs/config/config.go

```diff
@@ -73,9 +82,14 @@ func BuildEndpoints() (*client.Endpoints, error) {
 		main.Port = 443
 		useSSL = true
 	default:
-		// datadog settings
-		main.Host = coreConfig.Datadog.GetString("logs_config.dd_url")
-		main.Port = coreConfig.Datadog.GetInt("logs_config.dd_port")
+		// If no proxy is set, we default to 'logs_config.dd_url' if set, or to 'site'.
+		// if none of them is set, we default to the US agent endpoint.
+		main.Host = coreConfig.GetMainEndpoint(logsAgentPrefix, "logs_config.dd_url")
+		if port, found := logsEndpoints[main.Host]; found {
```

Comment:
what about `nossl` endpoints ?

Reply:
The `site` option allows for only proto endpoints with SSL. I think it's reasonable to ask. For no SSL endpoints you'll have to specify a `logs_config. dd_url` to `intake.logs.datadoghq.com` that will override the `site` config. And the SSL handling is left unchanged